tests/Fixtures/CustomArray.php

```diff
@@ -39,6 +39,7 @@ public function offsetExists($offset) {
      * @return mixed Can return all value types.
      * @since 5.0.0
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Had to check doc for this one, but all good. :)

Reply:
Yeah it's needed for the newer PHP versions because they declare a `mixed` return type that we can't add because it's not supported in PHP 8. Luckily annotations like this are treated as comments in old PHP versions.